src/main/java/org/folio/bulkops/service/Marc21ReferenceProvider.java

```diff
@@ -569,6 +570,7 @@ public class Marc21ReferenceProvider {
     languages.put("zxx", "No linguistic content");
     languages.put("zza", "Zaza");
 
+    mappedFields.put("041", INSTANCE_LANGUAGES);
```

Comment:
how this change is connected to this ticket?

Reply:
fixed